### ISSUE TITLE
More clear failure mechanism

### ISSUE THREAD

**Issue by davidtheclark** [necolas/postcss-bem-linter#44]
Per discussion in #26

**Comment by davidtheclark** [comment]
@necolas I will try to clarify the error in postcss-log-warnings. That would address one of your concerns, but not the duplicated printing of the warnings. I don't know --- I guess that ultimately we'd want the postcss runners to either be capable of throwing errors themselves or have an option *not* to print warnings, because something like postcss-log-warnings will handle it?

(Any thoughts on this @ai and @MoOx?)

**Comment by davidtheclark** [comment]
@necolas Please try postcss-log-warnings version v0.1.3 (just released) and see if you think it's more clear. I just added a message before the process exits saying "PostCSS warnings were logged".

**Comment by ai** [comment]
@davidtheclark I think `postcss-log-warnings` should clean `result.warnings()` if it already process all this warnings.

In this case we can do some magic like:

```
.use( linter() )
.use( logWarnings() )
.use( mixin() )
.use( messages() )
```

Here linter warnigns will be showed in log, but mixins warnings in page by postcss-messages.

**Comment by davidtheclark** [comment]
This is a fantastic idea, I think. I'll implement it now.

**Comment by davidtheclark** [comment]
This is done in postcss-log-warnings v0.2.0

**Comment by davidtheclark** [comment]
I guess any further discussion of this belongs in postcss-log-warnings.
